tests/AppIntegrationTest.php

```diff
@@ -235,6 +235,19 @@ static function ($request) {
         $this->assertSame('second', $this->client->get('/')->getBody()->getContents());
     }
 
+    public function testServerLogsAreNotInErrorOutput()
+    {
+        $expectedErrorOutput = '[Sun Feb 23 10:18:43 2020] [::1]:55146 [404]: (null) / - No such file or directory';
+        $serverLogsExample = '[Sun Feb 23 10:18:31 2020] PHP 7.4.2 Development Server (http://localhost:8086) started' . PHP_EOL .
+                             '[Sun Feb 23 10:18:43 2020] [::1]:55146 Accepted' . PHP_EOL .
+                             $expectedErrorOutput . PHP_EOL .
+                             '[Sun Feb 23 10:18:43 2020] [::1]:55146 Closing' . PHP_EOL;
+
+        $cleanedServerLogs = Server::cleanErrorOutput($serverLogsExample);
```

Comment:
Instead of make the method public and static I would suggest "injecting" lines the by [addErrorOutput()](https://github.com/symfony/symfony/blob/v4.4.4/src/Symfony/Component/Process/Process.php#L951) or [addOutput()](https://github.com/symfony/symfony/blob/v4.4.4/src/Symfony/Component/Process/Process.php#L937). If the Server would be a proxy for Process (I dont know if this would work...) the typical approach would be to mock the process. With inheritance of Server -> Process the addErrorOutput() method would be similar.

Reply:
Great idea, I didn't thought of doing this way. Thanks!